site/src/_pages/docs/2_publishing-blocks.mdx

```diff
@@ -30,14 +30,12 @@ The JSON metadata file should look like this:
 }
 ```
 
-<!-- markdownlint-disable no-emphasis-as-heading -->
```

Comment:
MDX 2 requires comments to be in `{* *}` instead of `<!-- -->`, which Prettier does on like. Upstream issue: https://github.com/prettier/prettier/issues/12209

In the meantime, if we do need to disable a markdownlint rule, we can do that in `.markdownlint.json`. I switched to regular MD headers here to avoid that. The text is a bit bigger than before but the doc looks good overall.